.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+A clear and concise description of what the bug is.
+
+## To Reproduce
+Please provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+
+>    …Minimal – Use as little code as possible that still produces the same problem
+>   …Complete – Provide all parts someone else needs to reproduce your problem in the question itself
+>   …Reproducible – Test the code you're about to provide to make sure it reproduces the problem
+ 
+
+## Expected behavior
+A clear and concise description of what you expected to happen.
+
+## Screenshots
+If applicable, add screenshots to help explain your problem.
+
+# Environment (please complete the following information):
+
+## Java/JDK
+`java -version`
+
+## Groovy version
+>__Note that versions older than 2.5 are no longer supported.__
+
+`groovy -version`
+
+## Build tool version
+
+### Gradle
+`gradle -version`
+
+### Apache Maven
+`mvn -version`
+
+## Operating System
+> Linux, Windows, Mac etc.
+
+## IDE
+> IntelliJ, Eclipse etc.
+
+## Build-tool dependencies used
+
+### Gradle/Grails
+    implementation 'org.spockframework:spock-core:2.0-groovy-3.0'
+
+### Apache Maven
+    <dependency>
+      <groupId>org.spockframework</groupId>
+      <artifactId>spock-core</artifactId>
+      <version>2.0-groovy-3.0</version>
+    </dependency>
+
+
+**Additional context**
```

Comment:
Why a bold text here instead of also a heading?
And if intentional maybe add a blank line after.